gulpfile.babel.js

```diff
@@ -18,8 +18,8 @@ import { getManagerHeadHtml, getPreviewBodyHtml, getPreviewHeadHtml } from './ut
 
 const browserSync = create();
 
-const build = series(generateTheme, components, copy, pack);
-const start = series(cleanAll, build, managerStream, webpackStream, server, watches, sbWatch);
+const build = series(cleanAll, generateTheme, components, copy, pack, staticServer);
```

Comment:
Build runs on the watches task, so it runs every time a changes made on component. We don't need staticServer build on changes. Probably should have a separate task for build static server